### PR TITLE
[Refactor] Code review fixes

### LINKED ISSUE #135: Small fixes
- [ ] https://github.com/EyeSeeTea/metadata-synchronization/pull/130#discussion_r304895535

Maybe add a link to documentation about the format of the cron expression? In vaccination-app we use react-linkify to render links easily, but feel free to do it how you see fit (if we really need a link here!)

- [x] https://github.com/EyeSeeTea/metadata-synchronization/pull/130#discussion_r304899425

You said in the description that booleans cannot be used, and indeed there is no type boolean in the UI of Dhis2. But is this limitation real? at least in user data store I am using booleans with no problems.

### PR DESCRIPTION
**Pull request by SferaDev**
### :pushpin: References

* **Issue:** Closes #135 

### :memo: Implementation

- Open "Edit Sync Rule" in a new page
- Add help to the "Frequency" step of the Sync Rule
- Add descriptive name for enabled on "Save Step"

### :art: Screenshots

image

image

### :bookmark_tabs: Others

- Any change in the [GUI library](https://github.com/EyeSeeTea/d2-ui-components)? If so, what branch/PR?

### CODE REVIEW

File: src/components/rules-creation-page/steps/SchedulerStep.jsx [hunk 1]
```diff
@@ -31,6 +21,14 @@ const Toggle = ({ label, onChange, value }) => (
 );
 
 const SchedulerStep = ({ syncRule, onChange }) => {
+    const cronExpressions = [
+        { text: i18n.t("Every hour"), value: "0 0 * ? * *" },
```
Comment (tokland) [inline src/components/rules-creation-page/steps/SchedulerStep.jsx]: Cool. I don't really understand why the dhis2 team had `apiTranslations.js`, then you end up with two parallel i18n infrastructures...

File: src/components/rules-creation-page/steps/SaveStep.jsx [hunk 2]
```diff
@@ -107,7 +105,10 @@ const SaveStep = ({ d2, syncRule, classes, onCancel, snackbar }) => {
                     </ul>
                 </LiEntry>
 
-                <LiEntry label={i18n.t("Scheduling enabled")} value={syncRule.enabled.toString()} />
+                <LiEntry
+                    label={i18n.t("Scheduling")}
+                    value={syncRule.enabled === "true" ? i18n.t("Enabled") : i18n.t("Disabled")}
```
Comment (tokland) [inline src/components/rules-creation-page/steps/SaveStep.jsx]: `syncRule.enabled` already return a boolean, doesn't it?

File: i18n/en.pot [hunk 3]
```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2019-07-16T15:14:24.210Z\n"
-"PO-Revision-Date: 2019-07-16T15:14:24.210Z\n"
+"POT-Creation-Date: 2019-07-18T20:18:16.329Z\n"
+"PO-Revision-Date: 2019-07-18T20:18:16.329Z\n"
```
Comment (tokland) [inline i18n/en.pot]: Is this file up-to-date? When I run `yarn extract-pot`, the file has some changes. (for example, it adds literal "Deleting Notifications").

**Comment by SferaDev** [comment]
Not sure about adding a link here, I believe we have enough information now about cron expressions

**Pull request merged**